### PR TITLE
feat: update registry vaults status

### PR DESCRIPTION
**Pull request by sajanrajdev**
Tackles issue #681 

Updates the RegistryV2 vaults' status to the current state in preparation to its imminent wiring to the UI.

Ethereum:
```
brownie run scripts/issue/681/update_registry_vault_status.py
```
Adds Aura vaults to experimental (and open for graviAURA) and deprecates both Digg vaults.

Fantom:
```
brownie run scripts/issue/681/update_registry_vault_status.py --network ftm-main-fork
```
Deprecates everything still active except for bveOXD and bveOXD LP

### CODE REVIEW

**Comment by sajanrajdev** [comment]
Ethereum call and events trace:
```
Running 'scripts/issue/681/update_registry_vault_status.py::main'...
['0x37d9D2C6035b744849C15F1BFEE8F268a20fCBd8', 'v1.5', 'name=auraBAL,protocol=Aura,behavior=Ecosystem Helper', 1]
Transaction sent: 0x6dbf5f619e40236e570cc8ab0b7c06d313432239b7abcc45f6eaceba836ba7b1
  Gas price: 0.0 gwei   Gas limit: 12000000   Nonce: 1
  IBadgerRegistryV2.promote confirmed   Block: 15213635   Gas used: 245157 (2.04%)

Promote: 0x37d9D2C6035b744849C15F1BFEE8F268a20fCBd8 (v1.5) to 1
['0x06D756861De0724FAd5B5636124e0f252d3C1404', 'v1.5', 'name=bb-a-USD,protocol=Aura,behavior=Ecosystem Helper', 1]
Transaction sent: 0x55284e31a8f178d69a8c482e5d354fadbf3e31e2d50e092472d664b52907b759
  Gas price: 0.0 gwei   Gas limit: 12000000   Nonce: 2
  IBadgerRegistryV2.promote confirmed   Block: 15213636   Gas used: 230386 (1.92%)

Promote: 0x06D756861De0724FAd5B5636124e0f252d3C1404 (v1.5) to 1
['0xf8f5677B6bCecdb9be94AE8f6770a05a6C53C378', 'v1.5', 'name=graviAURA/auraBAL/WETH,protocol=Aura,behavior=None', 1]
Transaction sent: 0x3ef1b06a2c325f415b33c98a6240dccea9f6a964492282bd299f0d71869ed3ad
  Gas price: 0.0 gwei   Gas limit: 12000000   Nonce: 3
  IBadgerRegistryV2.promote confirmed   Block: 15213637   Gas used: 233448 (1.95%)

Promote: 0xf8f5677B6bCecdb9be94AE8f6770a05a6C53C378 (v1.5) to 1
['0x371B7C451858bd88eAf392B383Df8bd7B8955d5a', 'v1.5', 'name=graviAURA/DIGG/WBTC,protocol=Aura,behavior=None', 1]
Transaction sent: 0x9f61c9a51d703952a1b0100d8511b52712ecaeea01f2a2b1f27f0e32e467c8ca
  Gas price: 0.0 gwei   Gas limit: 12000000   Nonce: 4
  IBadgerRegistryV2.promote confirmed   Block: 15213638   Gas used: 232761 (1.94%)

Promote: 0x371B7C451858bd88eAf392B383Df8bd7B8955d5a (v1.5) to 1
['0x63ad745506BD6a3E57F764409A47ed004BEc40b1', 'v1.5', 'name=BADGER/WBTC,protocol=Aura,behavior=None', 1]
Transaction sent: 0x31f065911075412aa90c4cbaf3d4f01738ed1b589c9e8b2489b39ba3ba5edfdc
  Gas price: 0.0 gwei   Gas limit: 12000000   Nonce: 5
  IBadgerRegistryV2.promote confirmed   Block: 15213639   Gas used: 230917 (1.92%)

Promote: 0x63ad745506BD6a3E57F764409A47ed004BEc40b1 (v1.5) to 1
['0xBA485b556399123261a5F9c95d413B4f93107407', 'v1.5', 'name=graviAURA,protocol=Aura,behavior=None', 3]
Transaction sent: 0x43792eb1bf646cfc4b4e688d03498547321f558a36beebf9bb71f512773cc9bd
  Gas price: 0.0 gwei   Gas limit: 12000000   Nonce: 6
  IBadgerRegistryV2.promote confirmed   Block: 15213640   Gas used: 248829 (2.07%)

Promote: 0xBA485b556399123261a5F9c95d413B4f93107407 (v1.5) to 3
['0x88128580ACdD9c04Ce47AFcE196875747bF2A9f6', 'v1', 'name=wBTC/Digg,protocol=Sushiswap,behavior=None', 0]
Transaction sent: 0x6aac0fc2ed1f0596cda3feb74a86ac99143d98b5adb14f37b4664bcb52f1f1bd
  Gas price: 0.0 gwei   Gas limit: 12000000   Nonce: 7
  IBadgerRegistryV2.demote confirmed   Block: 15213641   Gas used: 78873 (0.66%)

Demote: 0x88128580ACdD9c04Ce47AFcE196875747bF2A9f6 to 0
['0x7e7E112A68d8D2E221E11047a72fFC1065c38e1a', 'v1', 'name=Digg,protocol=Badger,behavior=None', 0]
Transaction sent: 0xa5211bf2e368c796b3f017d92765b2598400626aaf6cf76567a425a8c6580395
  Gas price: 0.0 gwei   Gas limit: 12000000   Nonce: 8
  IBadgerRegistryV2.demote confirmed   Block: 15213642   Gas used: 78873 (0.66%)

Demote: 0x7e7E112A68d8D2E221E11047a72fFC1065c38e1a to 0
Transaction sent: 0x492c79a244ffdd37f3ddec4207b96155b7e98a3a3461443d184947d875223aaf
  Gas price: 0.0 gwei   Gas limit: 12000000   Nonce: 27
  Gnosis Safe.approveHash confirmed   Block: 15213635   Gas used: 46206 (0.39%)

Transaction sent: 0xecdfa9a491353ac82c76144c758c431e914285731a9ae3033a36da007c7f4113
  Gas price: 0.0 gwei   Gas limit: 12000000   Nonce: 18
  Gnosis Safe.approveHash confirmed   Block: 15213636   Gas used: 46206 (0.39%)

Transaction sent: 0x100dc44b57b202c7d63fb833329bb2330b6e6706d0586badf82c4990672df2b2
  Gas price: 0.0 gwei   Gas limit: 12000000   Nonce: 3
  Gnosis Safe.approveHash confirmed   Block: 15213637   Gas used: 46206 (0.39%)

Transaction sent: 0x11590d6fb9827ab33a878f005e543ae288064c0a13e0b71319a88ab8ccc2a440
  Gas price: 0.0 gwei   Gas limit: 1625789   Nonce: 28
  Gnosis Safe.execTransaction confirmed   Block: 15213638   Gas used: 1429043 (87.90%)

Transaction was Mined
---------------------
Tx Hash: 0x11590d6fb9827ab33a878f005e543ae288064c0a13e0b71319a88ab8ccc2a440
From: 0x0a9af7FAba0d5DF7A8C881e1B9cd679ee07Af8A2
To: 0x86cbD0ce0c087b482782c181dA8d191De18C8275
Value: 0
Function: Gnosis Safe.execTransaction
Block: 15213638
Gas Used: 1429043 / 1625789 (87.9%)

Events In This Transaction
--------------------------
├── IBadgerRegistryV2 (0xdc602965F3e5f1e7BAf2446d5564b407d5113A06)
│   ├── PromoteVault
│   │   ├── author: 0x86cbD0ce0c087b482782c181dA8d191De18C8275
│   │   ├── version: v1.5
│   │   ├── metadata: name=auraBAL,protocol=Aura,behavior=Ecosystem Helper
│   │   ├── vault: 0x37d9D2C6035b744849C15F1BFEE8F268a20fCBd8
│   │   └── status: 1
│   ├── PromoteVault
│   │   ├── author: 0x86cbD0ce0c087b482782c181dA8d191De18C8275
│   │   ├── version: v1.5
│   │   ├── metadata: name=bb-a-USD,protocol=Aura,behavior=Ecosystem Helper
│   │   ├── vault: 0x06D756861De0724FAd5B5636124e0f252d3C1404
│   │   └── status: 1
│   ├── PromoteVault
│   │   ├── author: 0x86cbD0ce0c087b482782c181dA8d191De18C8275
│   │   ├── version: v1.5
│   │   ├── metadata: name=graviAURA/auraBAL/WETH,protocol=Aura,behavior=None
│   │   ├── vault: 0xf8f5677B6bCecdb9be94AE8f6770a05a6C53C378
│   │   └── status: 1
│   ├── PromoteVault
│   │   ├── author: 0x86cbD0ce0c087b482782c181dA8d191De18C8275
│   │   ├── version: v1.5
│   │   ├── metadata: name=graviAURA/DIGG/WBTC,protocol=Aura,behavior=None
│   │   ├── vault: 0x371B7C451858bd88eAf392B383Df8bd7B8955d5a
│   │   └── status: 1
│   ├── PromoteVault
│   │   ├── author: 0x86cbD0ce0c087b482782c181dA8d191De18C8275
│   │   ├── version: v1.5
│   │   ├── metadata: name=BADGER/WBTC,protocol=Aura,behavior=None
│   │   ├── vault: 0x63ad745506BD6a3E57F764409A47ed004BEc40b1
│   │   └── status: 1
│   ├── PromoteVault
│   │   ├── author: 0x86cbD0ce0c087b482782c181dA8d191De18C8275
│   │   ├── version: v1.5
│   │   ├── metadata: name=graviAURA,protocol=Aura,behavior=None
│   │   ├── vault: 0xBA485b556399123261a5F9c95d413B4f93107407
│   │   └── status: 3
│   ├── DemoteVault
│   │   ├── author: 0x86cbD0ce0c087b482782c181dA8d191De18C8275
│   │   ├── version: v1
│   │   ├── metadata: name=wBTC/Digg,protocol=Sushiswap,behavior=None
│   │   ├── vault: 0x88128580ACdD9c04Ce47AFcE196875747bF2A9f6
│   │   └── status: 0

│   └── DemoteVault
│       ├── author: 0x86cbD0ce0c087b482782c181dA8d191De18C8275
│       ├── version: v1
│       ├── metadata: name=Digg,protocol=Badger,behavior=None
│       ├── vault: 0x7e7E112A68d8D2E221E11047a72fFC1065c38e1a
│       └── status: 0
│
└── Gnosis Safe (0x86cbD0ce0c087b482782c181dA8d191De18C8275)
    └── ExecutionSuccess
        ├── txHash: 0xd874f59fe4c9973b72d26ad7ee4473d8d8f14cc5e5da14d453151db915fb01bf
        └── payment: 0
```

**Comment by sajanrajdev** [comment]
Fantom call trace (having an RPC issue on the mutlisend portion of the tx posting):
```
Running 'scripts/issue/681/update_registry_vault_status.py::main'...
['0xB6D63A4e5Ca740e96C26AdAbcAc73BE78Ee39Dc5', 'v1', 'name=WBTC/renBTC,protocol=Solidex,behavior=Compounder', 0]
Transaction sent: 0xd59558ce691e5605344c478ca69052f91f995383abd0d59a70dcd9240ce50daa
  Gas price: 0.0 gwei   Gas limit: 12000000   Nonce: 1
  IBadgerRegistryV2.demote confirmed   Block: 43537772   Gas used: 78873 (0.66%)

Demote: 0xB6D63A4e5Ca740e96C26AdAbcAc73BE78Ee39Dc5 to 0
['0xd9770deC6fdA576450e66f0c441B6b8755F7184A', 'v1', 'name=WEVE/USDC,protocol=Solidex,behavior=Compounder', 0]
Transaction sent: 0x5af829e66f3b6358faf2c5ecb07385c8b215946adeede7a983726d2f6341a89d
  Gas price: 0.0 gwei   Gas limit: 12000000   Nonce: 2
  IBadgerRegistryV2.demote confirmed   Block: 43537773   Gas used: 78873 (0.66%)

Demote: 0xd9770deC6fdA576450e66f0c441B6b8755F7184A to 0
['0x4145a9B8B7bdC76F14e65c26f9c71e23B9069b25', 'v1', 'name=WFTM/CRV,protocol=Solidex,behavior=Bitcoin Rewards', 0]
Transaction sent: 0xd6a7b346d265454e0ed81f02a414d2b00ffe53e765faf149187eeabda282b553
  Gas price: 0.0 gwei   Gas limit: 12000000   Nonce: 3
  IBadgerRegistryV2.demote confirmed   Block: 43537774   Gas used: 66185 (0.55%)

Demote: 0x4145a9B8B7bdC76F14e65c26f9c71e23B9069b25 to 0
['0x1AaF7f8154704D80E2380B3DbC967FD0486016e0', 'v1', 'name=USDC/MIM,protocol=Solidex,behavior=Bitcoin Rewards', 0]
Transaction sent: 0x358659390cf09d1d02e5239dc39c7ff2bf56220a91d16fd99b30a204fb0390ec
  Gas price: 0.0 gwei   Gas limit: 12000000   Nonce: 4
  IBadgerRegistryV2.demote confirmed   Block: 43537775   Gas used: 66185 (0.55%)

Demote: 0x1AaF7f8154704D80E2380B3DbC967FD0486016e0 to 0
['0x9a28eeFB2c5F8311f37c39314D78Be85C54B5da6', 'v1', 'name=WFTM/renBTC,protocol=Solidex,behavior=Bitcoin Rewards', 0]
Transaction sent: 0x26183e18cca491074d4702a3403e4bb165adaf33ee2fcd3ed8eac9671028562f
  Gas price: 0.0 gwei   Gas limit: 12000000   Nonce: 5
  IBadgerRegistryV2.demote confirmed   Block: 43537776   Gas used: 55593 (0.46%)

Demote: 0x9a28eeFB2c5F8311f37c39314D78Be85C54B5da6 to 0
['0xa8bD8655A0dCABE76913D821Ab437562276b3B59', 'v1.5', 'name=boxSOLID,protocol=0xDAO,behavior=Ecosystem Helper', 0]
Transaction sent: 0xd6818d00cbac0b2aa23bfdd83b578db9e9e05fcfb2eeff74b1c570371975e01b
  Gas price: 0.0 gwei   Gas limit: 12000000   Nonce: 6
  IBadgerRegistryV2.demote confirmed   Block: 43537777   Gas used: 93873 (0.78%)

Demote: 0xa8bD8655A0dCABE76913D821Ab437562276b3B59 to 0
['0xcd1e62b390373fcFeA87Dd06E2497Fdc907935fA', 'v1.5', 'name=USDC/DEI,protocol=0xDAO,behavior=Ecosystem', 0]
Transaction sent: 0xcdbde1e9b9dc82c9a10b330341a61a3c3578105b29446ad72f50ca467e7747f0
  Gas price: 0.0 gwei   Gas limit: 12000000   Nonce: 7
  IBadgerRegistryV2.demote confirmed   Block: 43537778   Gas used: 78873 (0.66%)

Demote: 0xcd1e62b390373fcFeA87Dd06E2497Fdc907935fA to 0
```

**Comment by petrovska-petro** [comment]
> Fantom call trace (having an RPC issue on the mutlisend portion of the tx posting):
> 
> ```
> Running 'scripts/issue/681/update_registry_vault_status.py::main'...
> ['0xB6D63A4e5Ca740e96C26AdAbcAc73BE78Ee39Dc5', 'v1', 'name=WBTC/renBTC,protocol=Solidex,behavior=Compounder', 0]
> Transaction sent: 0xd59558ce691e5605344c478ca69052f91f995383abd0d59a70dcd9240ce50daa
>   Gas price: 0.0 gwei   Gas limit: 12000000   Nonce: 1
>   IBadgerRegistryV2.demote confirmed   Block: 43537772   Gas used: 78873 (0.66%)
> 
> Demote: 0xB6D63A4e5Ca740e96C26AdAbcAc73BE78Ee39Dc5 to 0
> ['0xd9770deC6fdA576450e66f0c441B6b8755F7184A', 'v1', 'name=WEVE/USDC,protocol=Solidex,behavior=Compounder', 0]
> Transaction sent: 0x5af829e66f3b6358faf2c5ecb07385c8b215946adeede7a983726d2f6341a89d
>   Gas price: 0.0 gwei   Gas limit: 12000000   Nonce: 2
>   IBadgerRegistryV2.demote confirmed   Block: 43537773   Gas used: 78873 (0.66%)
> 
> Demote: 0xd9770deC6fdA576450e66f0c441B6b8755F7184A to 0
> ['0x4145a9B8B7bdC76F14e65c26f9c71e23B9069b25', 'v1', 'name=WFTM/CRV,protocol=Solidex,behavior=Bitcoin Rewards', 0]
> Transaction sent: 0xd6a7b346d265454e0ed81f02a414d2b00ffe53e765faf149187eeabda282b553
>   Gas price: 0.0 gwei   Gas limit: 12000000   Nonce: 3
>   IBadgerRegistryV2.demote confirmed   Block: 43537774   Gas used: 66185 (0.55%)
> 
> Demote: 0x4145a9B8B7bdC76F14e65c26f9c71e23B9069b25 to 0
> ['0x1AaF7f8154704D80E2380B3DbC967FD0486016e0', 'v1', 'name=USDC/MIM,protocol=Solidex,behavior=Bitcoin Rewards', 0]
> Transaction sent: 0x358659390cf09d1d02e5239dc39c7ff2bf56220a91d16fd99b30a204fb0390ec
>   Gas price: 0.0 gwei   Gas limit: 12000000   Nonce: 4
>   IBadgerRegistryV2.demote confirmed   Block: 43537775   Gas used: 66185 (0.55%)
> 
> Demote: 0x1AaF7f8154704D80E2380B3DbC967FD0486016e0 to 0
> ['0x9a28eeFB2c5F8311f37c39314D78Be85C54B5da6', 'v1', 'name=WFTM/renBTC,protocol=Solidex,behavior=Bitcoin Rewards', 0]
> Transaction sent: 0x26183e18cca491074d4702a3403e4bb165adaf33ee2fcd3ed8eac9671028562f
>   Gas price: 0.0 gwei   Gas limit: 12000000   Nonce: 5
>   IBadgerRegistryV2.demote confirmed   Block: 43537776   Gas used: 55593 (0.46%)
> 
> Demote: 0x9a28eeFB2c5F8311f37c39314D78Be85C54B5da6 to 0
> ['0xa8bD8655A0dCABE76913D821Ab437562276b3B59', 'v1.5', 'name=boxSOLID,protocol=0xDAO,behavior=Ecosystem Helper', 0]
> Transaction sent: 0xd6818d00cbac0b2aa23bfdd83b578db9e9e05fcfb2eeff74b1c570371975e01b
>   Gas price: 0.0 gwei   Gas limit: 12000000   Nonce: 6
>   IBadgerRegistryV2.demote confirmed   Block: 43537777   Gas used: 93873 (0.78%)
> 
> Demote: 0xa8bD8655A0dCABE76913D821Ab437562276b3B59 to 0
> ['0xcd1e62b390373fcFeA87Dd06E2497Fdc907935fA', 'v1.5', 'name=USDC/DEI,protocol=0xDAO,behavior=Ecosystem', 0]
> Transaction sent: 0xcdbde1e9b9dc82c9a10b330341a61a3c3578105b29446ad72f50ca467e7747f0
>   Gas price: 0.0 gwei   Gas limit: 12000000   Nonce: 7
>   IBadgerRegistryV2.demote confirmed   Block: 43537778   Gas used: 78873 (0.66%)
> 
> Demote: 0xcd1e62b390373fcFeA87Dd06E2497Fdc907935fA to 0
> ```

script and targeted vaults looks good !

i had the same issue locally for the ftm run, it stems from the fact that it seems to have hard time grabbing which is the next pending nonce on the safe, one way i think we could work around it is as following:

Screenshot 2022-07-26 at 12 04 08

managing to pass programmatically the nonce hard-coded may solve this issue, just hard-coding in the pic for the example ( for ftm techops next nonce will be 13), with that seems to go successfully to allow posting smoothly

**Comment by sajanrajdev** [comment]
> managing to pass programmatically the nonce hard-coded may solve this issue, just hard-coding in the pic for the example ( for ftm techops next nonce will be 13), with that seems to go successfully to allow posting smoothly

@petrovska-petro, great, will do that. Thanks!

File: scripts/issue/681/update_registry_vault_status.py [hunk 1]
```diff
@@ -0,0 +1,136 @@
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import get_registry as registry
+from brownie import chain
+
+contract_registry = registry()
+chain_safe = GreatApeSafe(contract_registry.badger_wallets.techops_multisig)
+chain_safe.init_badger()
+
+# Deprecating everything on Polygon except for bveOXD LP and bveOXD
```
Comment (axejintao) [inline scripts/issue/681/update_registry_vault_status.py:9]: nit: fantom

File: great_ape_safe/great_ape_safe.py [hunk 2]
```diff
@@ -252,7 +252,9 @@ def post_safe_tx(self, skip_preview=False, events=True, call_trace=False, reset=
         # events, call_trace and reset are params passed to .preview
         # silent=True: prevent printing of safe_tx attributes at end of run
         # post=True: make the actual live posting of the tx to the gnosis api
-        if not safe_tx:
+        if not safe_tx and replace_nonce:
+            safe_tx = self.multisend_from_receipts(safe_nonce=replace_nonce)
```
Comment (btcookies) [inline great_ape_safe/great_ape_safe.py:256]: shouldn't this be handled in L262 by setting safe_tx._safe_nonce?

**Review by axejintao — APPROVED** [review]
lgtm

File: scripts/issue/681/update_registry_vault_status.py (hunk 1 above)
Comment (sajanrajdev) [reply scripts/issue/681/update_registry_vault_status.py:9]: Ah, brain was ded. Thanks!

File: great_ape_safe/great_ape_safe.py (hunk 2 above)
Comment (sajanrajdev) [reply great_ape_safe/great_ape_safe.py:256]: This for an extra off case in which the transaction data can't be fetched either such as in this one.

**Pull request merged**